lib/project-data.ts

```diff
@@ -87,6 +88,27 @@ export class ProjectData implements IProjectData {
 		this.$errors.fail("No project found at or above '%s' and neither was a --path specified.", projectDir || this.$options.path || currentDir);
 	}
 
+	public getAppResourcesDirectoryPath(projectDir?: string): string {
+		if (!projectDir) {
+			projectDir = this.projectDir;
```

Comment:
if `initializeProjectData` has not been called, the `this.projectDir` will also be undefined and the path.join below will fail.

Reply:
Correct, I am unsure how to tackle this. The need for the additional `projectDir` parameter arose when using the getter during project creation. What would you recommend I check here?

Reply:
I would just return null here. The caller should decide what to do.